irohad/main/application.hpp

```diff
@@ -204,6 +204,8 @@ class Irohad {
    */
   virtual RunResult initWsvRestorer();
 
+  virtual std::shared_ptr<iroha::ametsuchi::Settings> initSettings();
```

Comment:
`initX` methods return `RunResult`, probably this should do something similar.
Maybe you can even avoid a separate method and call storage method in constructor directly.

Reply:
Fixed